src/frontend/src/components/Buttons.jsx

```diff
@@ -33,6 +34,11 @@ function ButtonsPanel({
       text: "Clear Annotation",
       onClick: onClearAnnotation,
     },
+    {
+      id: "download_annotation",
+      text: "Download Annotation",
+      onClick: onDownloadAnnotation,
```

Comment:
Just more consistency in naming as "Sample Annotation":
```suggestion
      id: "sample_annotation",
      text: "Sample Annotation",
      onClick: onSampleAnnotation,
```